src/node_messaging.cc

```diff
@@ -300,6 +333,16 @@ class SerializerDelegate : public ValueSerializer::Delegate {
     ThrowDataCloneException(context_, message);
   }
 
+  bool HasCustomHostObject(Isolate* isolate) override { return true; }
+
+  Maybe<bool> IsHostObject(Isolate* isolate, Local<Object> object) override {
+    if (BaseObject::IsBaseObject(object)) {
```

Comment:
Can we remove this branch by just appending it as a boolean expression to the next return statement?